package.json

```diff
@@ -23,45 +23,53 @@
         "Kontent.ai strongly typed models"
     ],
     "preferGlobal": true,
-    "exports": "./dist/es2022/lib/index.js",
-    "types": "./dist/es2022/lib/index.d.ts",
+    "exports": "./dist/es2022/lib/public_api.js",
+    "types": "./dist/es2022/lib/public_api.d.ts",
     "type": "module",
     "scripts": {
-        "release": "npm run lint:check && standard-version && npm run update:version",
-        "release:major": "npm run lint:check && standard-version --release-as major && npm run update:version",
-        "release:major:beta": "npm run lint:check && standard-version --prerelease --release-as major && npm run update:version",
+        "release": "standard-version && npm run update:version",
+        "release:major": "standard-version --release-as major && npm run update:version",
+        "release:major:beta": "standard-version --prerelease --release-as major && npm run update:version",
         "release:beta": "standard-version --prerelease",
         "prerelease:standard": "standard-version --prerelease",
         "prerelease:major": "standard-version --prerelease --release-as major",
-        "prepublishOnly": "npm run lint:check && npm run build && npm run push:tags",
+        "prepublishOnly": "npm run build && npm run push:tags",
         "publish:beta": "npm publish --tag=next",
         "build:es2022": "tsc --p tsconfig.es2022.json",
-        "build": "npm run clean && npm run build:es2022",
-        "test:all": "npm run build && npm run test:delivery-models",
-        "test:delivery-models": "tsx scripts/generate-delivery-models",
-        "test:help": "npm run build && cd sample && node ../dist/es2022/lib/cli/cli -h",
-        "lint:check": "npx eslint lib",
-        "clean": "tsx clean.ts",
+        "build": "npm run lint && npm run clean && npm run build:es2022",
```

Comment:
It would be a bit simpler, but I like the "visibility" it provides at a glance. We probably won't distribute multiple versions (unlike some SDKs do), but I don't really see a problem in making it a bit more organized and clear.

Reply:
I have to disagree here. It doesn't make anything more organized or clear, because when I see the `tsconfig.es2022.json` I expect more versions of `tsconfig`. I am confused and start searching for more versions and can't find any. It's like naming every file like `lib/fetch/kontent-fetcher.version-1.ts`, because in the future you might want to have a different version of the fetcher as well for different use cases. That's called premature optimization.

Reply:
Ok, fair enough. I wouldn't really categorize this as "optimization" though :D It was more for clarity at first glance. I don't mind having it the way you described so I modified the code to use only a single tsconfig and changed paths to remove the reference to "es2022"